modules/rosa-create-objects.adoc

```diff
@@ -122,13 +168,15 @@ $ rosa create cluster --cluster=<cluster_name> | <cluster_id> [arguments]
 |--region
 |The AWS region (string) where your worker pool will be located. This argument overrides the `AWS_REGION` environment variable.
 
+|--role-arn string
+|The Amazon Resource Name (ARN) of the role that {cluster-manager} will assume to create the cluster.
```

Comment:
I has to re-read this twice to understand what it meant. I was thinking "assume" was meaning that it will suppose to be the case, not that it will actually use. Could you say something like "use" instead of "assume" and have it mean the same thing?

Reply:
The OCM and the operators inherit the role based on the associated IAM policies and privileges. The word "assume" is used quite frequently in this context for AWS STS resource delegation and is included in the naming of the AWS API action set. Given this, I will leave it as is for now. For more details, see [AssumeRole](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html) in the AWS documentation.